src/styles/global.scss

```diff
@@ -0,0 +1,13 @@
+@import "./variables.scss";
+@import "./mixins.scss";
+@import "./card";
+@import "./stars";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
```

Comment:
hint: you can use `index.scss` to import all style files and keep the common styles for body, html, ... etc. separately in `main.scss`, `common.scss` or as you did in `global file scss.